tests/DataSet/ObjectDataSetTest.php

```diff
@@ -259,23 +257,17 @@ public function testValidateWithCallbackMethod(): void
         $this->assertSame(['name' => ['Value must be "foo"!']], $result->getErrorMessagesIndexedByPath());
     }
 
-    public function validateWithWrongCallbackMethodDataProvider(): array
-    {
-        return [
-            [new ObjectWithNonExistingCallbackMethod()],
-            [new ObjectWithNonPublicCallbackMethod()],
-            [new ObjectWithNonStaticCallbackMethod()],
-        ];
-    }
-
-    /**
-     * @link https://github.com/yiisoft/validator/issues/223
-     * @dataProvider validateWithWrongCallbackMethodDataProvider
```

Comment:
Why removed this tests?

Reply:
Because the functionality it implements became as a feature